src/js/components/replenishment/CreateReplenishment.jsx

```diff
@@ -261,8 +325,11 @@ class CreateReplenishment extends Component {
               <div className="table-form">
                 {_.map(FIELD, (fieldConfig, fieldName) =>
                   renderFormField(fieldConfig, fieldName, {
-                    updateRow: this.updateRow,
                     values,
+                    updateRow: this.updateRow,
+                    updateSelectedItems: this.updateSelectedItems,
+                    isAllProductsSelected: this.isAllProductsSelected(),
```

Comment:
If you need a helper function `allRowsSelected` in other places, then:
`allRowsSelected: this.allRowsSelected(values.requirements)`
and
```
allRowsSelected(rows) {
    return !_.find(rows, row => !row.checked)
}
```
otherwise:
`allRowsSelected: !_.find(values.requirements, row => !row.checked)`